tests/robot-tests/tests/admin/bau/create_publication_and_release.robot

```diff
@@ -26,7 +26,7 @@ Go to Create publication page for "UI tests topic" topic
 Enters contact details
     user enters text into element    id:publicationForm-teamName    Post-16 statistics team
     user enters text into element    id:publicationForm-teamEmail    post16.statistics@education.gov.uk
-    user enters text into element    id:publicationForm-contactName    Suzanne Wallace
+    user enters text into element    id:publicationForm-contactName    ui tests
```

Comment:
Rather than just `ui tests`, I would go with something that describes what it is - something like `UI Tests Contact Name`.